base/common/src/main/java/com/netscape/certsrv/util/JSONSerializer.java

```diff
@@ -1,19 +1,51 @@
 package com.netscape.certsrv.util;
 
+import com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.PropertyAccessor;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.module.jaxb.JaxbAnnotationIntrospector;
 
 /**
  * Implement this interface to provide default methods to serialize an object to/from JSON
  */
 public interface JSONSerializer {
 
+    default String toJSON(boolean requiresPrivateFieldAccess) throws Exception {
+        if (requiresPrivateFieldAccess) {
+            ObjectMapper mapper = new ObjectMapper();
+            return mapper.enable(SerializationFeature.INDENT_OUTPUT)
+                    .setVisibility(PropertyAccessor.FIELD, Visibility.ANY)
+                    .setAnnotationIntrospector(new JaxbAnnotationIntrospector(mapper.getTypeFactory()))
+                    .setSerializationInclusion(Include.NON_NULL)
```

Comment:
We might be able to drop `@JsonInclude(Include.NON_NULL)` from all classes.

Reply:
I removed `setSerializationInclusion(Include.NON_NULL)` from the `ObjectMapper` and also the annotations from the `Account` class and set all the fields to `null` - none of the null fields appear in the JSON and the tests pass so I guess that can all safely be removed.

Reply:
(I'll do that everywhere in one PR though else the diff gets big)

Reply:
Sure, thanks!